### PR TITLE
Sync wc description

### PR DESCRIPTION
**Pull request by daniel17903**
- description and synonyms can now be edited
- description and synonyms are synchronized to wc
- items can be found by searching for synonyms
- removed field "properties" (content was mostly similar to description)

LeihLokalVerwaltung             |  Website
:-------------------------:|:-------------------------:
image  |  image

### CODE REVIEW

File: Frontend/src/components/Input/PopupFormular.svelte [hunk 1]
```diff
@@ -1,166 +1,175 @@
...
-    import Checkbox from "svelte-checkbox";
-    import SelectInput from "./SelectInput.svelte";
-    import InputGroup from "./InputGroup.svelte";
-    import DateInput from "./DateInput.svelte";
-    import TextInput from "./TextInput.svelte";
-    import AutocompleteInput from "./AutocompleteInput.svelte";
-    import InputTypes from "./InputTypes";
-    import { keyValueStore } from "../../utils/stores";
-    import { createEventDispatcher } from "svelte";
+  import Checkbox from "svelte-checkbox";
```
Comment (skjerns) [inline Frontend/src/components/Input/PopupFormular.svelte:2]: I sometimes have the feeling that VSCode can't get it right and adds/removes whitespaces in a mood :D

**Review by skjerns — APPROVED** [review]
looks good!

**Comment by skjerns** [comment]
Do you think the white-space and formatting divs might be due to us having slightly different settings enabled in the autoformat? We should find out how we can sync our settings to make the diffs more clearly :)

good work! thanks :)

**Comment by daniel17903** [comment]
> Do you think the white-space and formatting divs might be due to us having slightly different settings enabled in the autoformat? We should find out how we can sync our settings to make the diffs more clearly :)
> 
> good work! thanks :)

I suspect that after you reformatted everything the VSCode plugin updated and now has some different default settings. You are right, we should use the same settings.

**Pull request merged**